tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -353,13 +353,26 @@ class CompletionSuite extends BaseCompletionSuite {
        |ProcessBuilder - scala.sys.process
        |CertPathBuilder - java.security.cert
        |CertPathBuilderSpi - java.security.cert
-       |ProcessBuilderImpl - scala.sys.process
```

Comment:
Interesting, seems it's a private trait, so it shouldn't show up for Scala 2 :thinking: 

Not important in this PR though.

Reply:
yes, Metals should not suggest private members. Room for investigation, later.

Reply:
I put it in an issue to later work on: https://github.com/scalameta/metals/issues/4206